README.md

```diff
@@ -50,6 +50,77 @@ During the installation process (if you follow the steps on [postgresql.org](htt
 #### Viewing/Editing the DB
 If you'd like a visual way of viewing or editing your local database, try using [TablePlus](https://tableplus.com).
 
+## Environment Variables
+
```

Comment:
Please add a little header text here to explain what this section covers and then add all the below to a `details` element like this:
``` html
<details>
  <summary>
    Local Environment Variable Configuration
  </summary>
  <!-- Your content here -->
</details>
```

You might need to convert some of the content to HTML, but try markdown first